python/rikai/spark/sql/generated/__init__.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright (c) 2021 Rikai Authors
```

Comment:
will this file be deleted after a new `make generated`?

Reply:
i added touch in the makefile

Reply:
nit, i am confused whether this file will be regererated via `make antrl`.
If yes, then the content in this file will be lost next time we run `make antlr`.
If no, we dont need to `touch` in Makefile?

Reply:
Locally it depends on whether you deleted the whole directory contents or just removed one file. If you just change the g4 file without touching the generated directory make will do nothing (at least on Ubuntu). So we can't really guarantee one way or the other but this at least ensures we don't run into the same situation where those python files don't get added to the wheel. Touch will do nothing if the file already exists so it's not an issue either.

Reply:
This file is not by default generated by the antlr command

Reply:
Init will have no contents other than the license. It's just to make that it's a python module to setup